cypress/integration/pages/onDemandAudio/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,77 @@
+import path from 'ramda/src/path';
+import envConfig from '../../../support/config/envs';
+import { overrideRendererOnTest } from '../../../support/helpers/onDemandRadioTv';
+import {
+  isScheduleDataComplete,
+  getIsProgramValid,
+} from '../../../../src/app/containers/RadioSchedule/utilities/evaluateScheduleData';
+
+export default ({ service, pageType, variant }) => {
+  describe('Chartbeat', () => {
+    if (envConfig.chartbeatEnabled) {
+      it('should have a script with src value set to chartbeat source', () => {
+        cy.hasScriptWithChartbeatSrc();
+      });
+      it('should have chartbeat config set to window object', () => {
+        cy.hasGlobalChartbeatConfig();
+      });
+    }
+  });
+  describe(`testsForCanonicalOnly for ${service} ${pageType} ${variant}`, () => {
+    beforeEach(() => {
+      cy.getToggles(service);
+    });
+
+    describe('Radio Schedule', () => {
```

Comment:
I think the radio schedule tests will now only be for OD radio, not podcasts... so this will need wrapping in a conditional. At the moment I think the `scheduleIsEnabled` toggle would evaluate to 'true' for services with them enabled, when on a podcast page?

I'd also be inclined to suggest you break out the Radio Schedule tests into its own file and import them here.

Reply:
Podcasts DO have schedules yeoooo. Ignore the above